packages/react/src/grouped-fields/docs/overview.mdx

```diff
@@ -92,10 +93,10 @@ When both inputs are required or one input is optional and the other has no empt
 						{ value: 'mg', label: 'mg' },
 						{ value: 'g', label: 'g' },
 						{ value: 'kg', label: 'kg' },
-						{ value: 'ton', label: 'ton' },
+						{ value: 't', label: 't' },
+						{ value: 'Mt', label: 'Mt' },
 					]}
 					maxWidth="sm"
-					value="kg"
```

Comment:
This is missing `placeholder="Choose"`

Reply:
@Dan503 Sorry, I should have exlplicitly called out that this doesn't have the `placeholder` because it's showcasing an optional field that has a pre-selected `value` (`mg` in this case as it's the first)

Reply:
It seems odd to me that there is inconsistency in the examples. if an example has multiple inputs, all the inputs in that example should either all have placeholder text or none have placeholder text.

Reply:
But it's showcasing a very specific example which is called out before it: When both inputs are required *or one input is optional and the other has no empty state*.

This is a perfectly valid scenario that we're providing an example for